pages/en-us/components/button.mdx

```diff
@@ -50,15 +50,19 @@ Used to trigger an operation.
 <Playground
   title="Types"
   desc="Different state."
-  scope={{ Button, Spacer }}
+  scope={{ Button, Spacer, Grid }}
   code={`
-<>
-  <Button type="secondary">Secondary</Button> <Spacer y={.5} />
-  <Button type="success">Success</Button> <Spacer y={.5} />
-  <Button type="warning">Warning</Button> <Spacer y={.5} />
-  <Button type="error">Error</Button> <Spacer y={.5} />
-  <Button type="abort">Abort</Button>
-</>
+<Grid.Container gap={2}>
+  <Grid><Button auto type="secondary">Secondary</Button></Grid>
+  <Grid><Button auto type="success">Success</Button></Grid>
+  <Grid><Button auto type="warning">Warning</Button></Grid>
+  <Grid><Button auto type="error">Error</Button></Grid>
+  <Grid><Button auto type="abort">Abort</Button></Grid>
+  <Grid><Button auto type="secondary-light">Secondary Ligth</Button></Grid>
```

Comment:
It should be ``Light``.

Reply:
Now the dilemma is that `light` can only be combined with 4 basic type.(`success-light` / `warning-light` ...) 
If `light` is a single prop, but it only works under 4 basic type, which seems strange 🤕

Reply:
I mean that you have a typo in the button texts (lines 61-64) - it should be ``Light``, not ``Ligth``.

Reply:
😿